graylog2-server/src/main/java/org/graylog2/rest/resources/cluster/ClusterInputStatesResource.java

```diff
@@ -100,10 +93,10 @@ public ClusterInputStatesResource(NodeService nodeService,
                         if (response.isSuccess()) {
                             return response.body().states();
                         } else {
-                            LOG.warn("Unable to fetch input states from node " + entry.getKey() + ": " + response.message());
+                            LOG.warn("Unable to fetch input states from node {}: {}", entry.getKey(), response.message());
                         }
                     } catch (IOException e) {
-                        LOG.warn("Unable to fetch input states from node " + entry.getKey() + ": ", e);
+                        LOG.warn("Unable to fetch input states from node {}: {}", entry.getKey(), e);
```

Comment:
Leaving out the last placeholder (`{}`) in the message would print the exception's stack trace instead of just calling the `toString()` method: http://www.slf4j.org/faq.html#paramException